deps/rabbit/src/rabbit_db_binding.erl

```diff
@@ -707,21 +708,14 @@ match_routing_key_in_mnesia(SrcName, RoutingKeys, UseIndex) ->
     end.
 
 match_routing_key_in_khepri(Src, ['_']) ->
-    try
-        MatchHead = #index_route{source_key  = {Src, '_'},
-                                 destination = '$1',
-                                 _           = '_'},
-        ets:select(?KHEPRI_INDEX_ROUTE_PROJECTION, [{MatchHead, [], ['$1']}])
-    catch
-        error:badarg ->
-            []
-    end;
+    ets:lookup_element(?KHEPRI_ROUTE_BY_SOURCE_PROJECTION,
+                       Src, #route_by_source.destination, []);
```

Comment:
I think we want to keep the `try`/`catch` here, see https://github.com/rabbitmq/rabbitmq-server/pull/11667 / https://github.com/rabbitmq/rabbitmq-server/pull/11700

Reply:
Thank you. Fixed.